test/workflow_test.go

```diff
@@ -860,6 +860,56 @@ func (w *Workflows) LargeQueryResultWorkflow(ctx workflow.Context) (string, erro
 	return "hello", nil
 }
 
+func (w *Workflows) MutatingQueryWorkflow(ctx workflow.Context) (string, error) {
+	err := workflow.SetQueryHandler(ctx, "mutating_query", func() (string, error) {
```

Comment:
Sorry I am just missing it, but where is the `setIsReadOnyl(true)` for query functions?

Reply:
it's not set because when queries are run the dispatcher isn't running so that will fail before we even check.